internal/dsl/dsl.go

```diff
@@ -0,0 +1,986 @@
+// Copyright 2024 The Serverless Workflow Specification Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package dsl
+
+const DSLSpec = `
+$id: https://serverlessworkflow.io/schemas/1.0.0-alpha1/workflow.yaml
```

Comment:
We can upgrade :D

Reply:
Maybe we can think of a way to update this automatically via CI. Once you open the issues to follow up on this PR, can you open this one? I can take care of it.